scripts/setup/configure-certs

```diff
@@ -0,0 +1,24 @@
+#!/bin/env bash
+
+# Realm-specific construct
+if [ -z "$RLMNAME" ]; then
+    echo "No Realm name given."
+    RLMNAME="A boson of realm"
+fi
```

Comment:
We might want to pick a clearer name for the variable.  I think we want it to be "SERVER_NAME", since technically the cert is for a server, not a realm.  Also as a general rule, we don't use abbreviations in Zulip naming.